src/mpid/ch4/netmod/ofi/ofi_recv.h

```diff
@@ -274,20 +274,13 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_mpi_cancel_recv(MPIR_Request * rreq)
     MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_MPIDI_NM_MPI_CANCEL_RECV);
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDI_NM_MPI_CANCEL_RECV);
 
-    if (!MPIDI_OFI_ENABLE_TAGGED) {
```

Comment:
Hmm. Good question. Well @charlesarcher isn't around anymore for us to ask, but I think you're right. I'll revert this part out.